src/plasoscaffolder/bll/services/base_sqlite_plugin_helper.py

```diff
@@ -55,3 +56,16 @@ def IsValidPluginName(self, plugin_name: str) -> bool:
     Returns:
       bool: true if the plugin name is valid
     """
+
+  @abc.abstractmethod
+  def RunSQLQuery(self, query: str,
+                  executor: base_sql_query_execution.BaseSQLQueryExecution()):
+    """ Validates the sql query
+
+    Args:
+      executor (base_sql_query_execution.SQLQueryExection()) the sql executor
+      query (str): the sql query 
+
+    Returns:
+      base_sql_query_execution.SQLQueryData: the data to the executed query
```

Comment:
I would suggest removing the module prefix for any modules in the scaffolder code base
```
SQLQueryData: data returned by executing the query
```